### DISCUSSION TITLE
Buttons

### DISCUSSION THREAD

**Discussion by ramonflores08** [OMOTE-Community/OMOTE-Firmware#13] [Ideas]
I have ordered all the parts from the BOM and noticed that the button price adds up quite a bit depending on where you order them. I was wondering if some silicone buttons would be both more cost-effective and simpler to build. Something like the link below, IDK the process of having a custom set of buttons would be but that would be ideal.

https://www.alibaba.com/product-detail/Factory-customize-silicone-single-point-push_62417902400.html?spm=a2700.7724857.0.0.68307ecc89iDrQ

![buttons](https://github.com/CoretechR/OMOTE/assets/7199615/6ace3a0c-4183-4612-9dee-d1caffff673c)

**Comment by CoretechR** [comment]
Sure, silicone buttons would be ideal for manufacturing at larger scale. But a custom keyboard needs special moulds.
I imagine that beyond 100 units or so, the silicone keypad start to make sense financially vs the tactile switches. I just liked that the switches can be sourced from anywhere and are not specific to this project.
